<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>fwbackups Server Administrator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="serverPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="serverNewButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>New...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="serverEditButton">
            <property name="text">
             <string>Edit...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="serverRemoveButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QListView" name="serversListView"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Server details</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Hostname:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Server is running:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Accounts:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="detailsHostnameLabel">
               <property name="text">
                <string>If you see this, report a bug</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="3">
              <widget class="QLabel" name="detailsAccountsLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="3">
              <widget class="QLabel" name="detailsRunningLabel">
               <property name="text">
                <string>No</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Certificate Authority:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="3">
              <widget class="QLabel" name="detailsCAlabel">
               <property name="text">
                <string>If you see this, report a bug</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Port:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLabel" name="detailsPortLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>2201</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="accountsPage">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Server hostname:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="accountAddButton">
              <property name="text">
               <string>Add account...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="accountEditButton">
              <property name="text">
               <string>Edit account...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="accountDeleteButton">
              <property name="text">
               <string>Delete account</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>5</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QListView" name="accountsListView"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="accountServerCombo"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="activeConnectionsPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Active connections listed by server:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="activeConnectionsTreeView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Updates"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionManage_Servers_2"/>
    <addaction name="actionManage_Accounts_2"/>
    <addaction name="actionActive_Connections_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Server Administration - fwbackups</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionManage_Servers"/>
   <addaction name="actionManage_Accounts"/>
   <addaction name="actionActive_Connections"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionManage_Servers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/preferences-system.png</normaloff>:/program-icons/program-icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Manage Servers</string>
   </property>
   <property name="toolTip">
    <string>Add, modify or remove servers and view their current status</string>
   </property>
  </action>
  <action name="actionActive_Connections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/network.png</normaloff>:/program-icons/program-icons/network.png</iconset>
   </property>
   <property name="text">
    <string>Active Connections</string>
   </property>
   <property name="toolTip">
    <string>View connections to the servers in real time</string>
   </property>
  </action>
  <action name="actionManage_Accounts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/user.png</normaloff>:/program-icons/program-icons/user.png</iconset>
   </property>
   <property name="text">
    <string>Manage Accounts</string>
   </property>
   <property name="toolTip">
    <string>Manage client accounts for each server</string>
   </property>
  </action>
  <action name="actionManage_Servers_2">
   <property name="text">
    <string>Manage Servers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionManage_Accounts_2">
   <property name="text">
    <string>Manage Accounts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionActive_Connections_2">
   <property name="text">
    <string>Active Connections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../fwbackups.qrc"/>
 </resources>
 <connections/>
</ui>
